inc/profiles.php

```diff
@@ -74,3 +74,18 @@ function after_setup_theme() {
 	add_image_size( 'responsive_profile_large', $width, $height, $crop );
 }
 add_action( 'after_setup_theme', __NAMESPACE__ . '\after_setup_theme' );
+
+if ( ! defined( 'BU_PROFILES_PLUGIN_ACTIVE' ) ) {
```

Comment:
Could `BU_PROFILES_PLUGIN_ACTIVE` ever be defined but set to `false`?

`if ( ! defined( 'BU_PROFILES_PLUGIN_ACTIVE' ) || ! BU_PROFILES_PLUGIN_ACTIVE )` would catch that scenario.

Reply:
Honestly I can't tell you if that is possible but I think it's good to be safe. Thank you so very much for your suggestion. I just added it.